app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -181,17 +187,66 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
                 Ok(views.html.account_already_exists())
             })
         }
+      },
+                 whenIneligible = { (ineligible, nSIUserInfo) ⇒
+        ineligible.manualCreationAllowed match {
```

Comment:
its an overkill to use pattern matching for booleans

Reply:
@Jackie1988 ^^^

Reply:
ok sure

Reply:
updated, thanks